docs/source/implementations/feature_importance.rst

```diff
@@ -0,0 +1,171 @@
+.. _implementations-feature_importance:
```

Comment:
Please move most of the contents of the documentation into the code docstrings and check my docs related to CV for an example on how to link them in the doc file. 

Also, the doc generator gives some important warnings which you should attend to.

```
mlfinlab/docs/source/implementations/feature_importance.rst:15: WARNING: Unknown target name: "feature_importances".
looking for now-outdated files... none found
pickling environment... done
checking consistency... mlfinlab/docs/source/implementations/feature_importance.rst: WARNING: document isn't included in any toctree
```